dashboard/lib/policies/lti.rb

```diff
@@ -69,13 +69,19 @@ module AccessTokenScopes
 
   MAX_COURSE_MEMBERSHIP = 650
 
-  def self.get_account_type(id_token)
-    id_token[LTI_ROLES_KEY].each do |role|
+  def self.get_account_type(roles)
+    roles.each do |role|
       return User::TYPE_TEACHER if TEACHER_ROLES.include? role
     end
     return User::TYPE_STUDENT
   end
 
+  def self.get_email(id_token)
```

Comment:
There is some [similar functionality](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/lib/services/lti.rb#L69-L71) implemented in `services/lti.rb` that can be used to fetch a claim from the JWT, either at the top-level or from the custom claims. If it's sufficient, we might not need to implement a separate function for retrieving email.